test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Error.Tests.ps1

```diff
@@ -125,4 +125,16 @@ Describe 'Get-Error tests' -Tag CI {
 
         $out | Should -BeLikeExactly "*$expectedExceptionType*"
     }
+
+    It 'Get-Error uses Error color for Message and PositionMessage members' -Skip:(!$EnabledExperimentalFeatures.Contains("PSAnsiRendering")) {
+
+        $out = pwsh -noprofile -command '$PSStyle.OutputRendering = "ANSI"; [System.Management.Automation.Internal.InternalTestHooks]::SetTestHook("BypassOutputRedirectionCheck", $true); try { 1/0 } catch { }; Get-Error' | Out-String
+
+        # need to escape the open square bracket so the regex works
+        $resetColor = $PSStyle.Reset.Replace('[','\[')
+        $errorColor = $PSStyle.Formatting.Error.Replace('[','\[')
+        $out | Should -Match "Message +: ${resetColor}${errorColor}[A-z]+"
+        # match the position message underline
+        $out | Should -Match ".*?${errorColor}~~~"
```

Comment:
Should it be multiline Regex-s?

Reply:
I tried that originally, but I don't think Pester supports it.  I could strip the newlines for a more precise match, but in this case the 3 tildes is pretty specific which underlines `1/0`.